app/services/imports/sales_logs_import_service.rb

```diff
@@ -277,7 +278,8 @@ def fields_not_present_in_softwire_data
          hodate_check
          saledate_check
          student_not_child_value_check
-         discounted_sale_value_check]
+         discounted_sale_value_check
+         buyer_livein_value_check]
```

Comment:
It's not relevant to this ticket directly but I'm not sure why softwire is in the name of any variables!

Reply:
I think it's cause maybe we were getting the import data from a Softwire team at the time. I'm not sure, it is a bit misleading

Reply:
do we want to rename it? what is this list of attributes?
I'm guessing since it's a list of value checks then it's actually attributes not present in old core or dluhc data?